lib/msal-core/test/telemetry/DefaultEvent.spec.ts

```diff
@@ -15,7 +15,10 @@ describe("DefaultEvent", () => {
             sdk: "javascript",
             sdkVersion: "1.0.0",
             applicationName: "mochaTest",
-            applicationVersion: "1.22"
+            applicationVersion: "1.22",
+            networkInformation: {
+                connectionSpeed: "4g"
```

Comment:
Can "4g" be an enum?

Reply:
Since this is a string coming from the browser and we do nothing programatically with it other than dump it in telemetry I opted not to.